### PR TITLE
nodejs week3

### PR DESCRIPTION
**Pull request by AnilMula**

### CODE REVIEW

File: nodejs/week3/mealshare/src/backend/api/meals.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        .limit(request.query.limit)
+        .then((data) => response.json(data));
+    }
+  } catch (error) {
+    response.status(404).send("Bad request").end();
+    console.log(error);
+  }
+});
+
+// 2.adds a new meal
```
Comment (Laila1222) [inline nodejs/week3/mealshare/src/backend/api/meals.js:63]: Well done! 👍

File: nodejs/week3/mealshare/src/backend/api/meals.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+const { json } = require("body-parser");
+
+//http://localhost:3000/api/meals
+//1.returns all meals
+router.get("/", async (request, response) => {
```
Comment (Laila1222) [inline nodejs/week3/mealshare/src/backend/api/meals.js:8]: Great work!

**Review by Laila1222 — APPROVED** [review]
Awesome work! 💯 
See some small issues below.

File: nodejs/week3/calculator/calculator/calculator.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+const router = express.Router();
+const { request, response } = require("express");
+
+router.get("/", (request, response) => {
+  response.send("Basic calculator");
+});
+
+router.get("/add", async (request, response) => {
+  const result =
+    Number(request.query.firstParam) + Number(request.query.secondParam);
```
[the remark this answers is not in the document]
Comment (AnilMula) [reply nodejs/week3/calculator/calculator/calculator.js:11]: implemented

File: nodejs/week3/calculator/calculator/calculator.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    Number(request.query.firstParam) - Number(request.query.secondParam);
+
+  response.send(
+    `substraction of ${request.query.firstParam} and ${request.query.secondParam} is ${result}`
+  );
+});
+
+router.get("/multiply", (request, response) => {
+  const result =
+    Number(request.query.firstParam) * Number(request.query.secondParam);
```
[the remark this answers is not in the document]
Comment (AnilMula) [reply nodejs/week3/calculator/calculator/calculator.js:29]: implemented

**Pull request merged**